Base/usr/share/man/man7/kernel_boot_parameters.md

```diff
@@ -0,0 +1,142 @@
+## Name
+
+Kernel Boot Parameters - strings recognized by the kernel, that determine the
```

Comment:
This is technically called "kernel cmdline", so the man page should be named `cmdline`.

And the description should be shorter, how about "cmdline - kernel boot arguments"?

Reply:
I'm suggesting to call the file `kernel_cmdline.md`. What do you think?

Also, I suggest to have a description like this:

> Kernel cmdline - boot arguments to control the kernel functionality

Reply:
"kernel_cmdline" will be more descriptive, but then it doesn't quite sound like a Unix manpage name, whereas "cmdline" sounds extremely like one. OTOH maybe it's OK, there's "path_resolution" in Linux manpages after all.

> Kernel cmdline - boot arguments to control the kernel functionality

Fine, but:
* Drop "the"
* It's not just kernel functionality that it controls, userspace will also read `/proc/cmdline` and react to it. SystemServer does already.

Reply:
There's already a file called `kernel_modules.md` in the same folder, so I don't think there's a problem with such naming.

About the description, how about this:
> Kernel cmdline - boot arguments to control the system's functionality

I "insist" on "the", because I don't think it's correct to write `control system's functionality` here. I might be wrong about this.

Reply:
> There's already a file called kernel_modules.md in the same folder, so I don't think there's a problem with such naming.

Fair point

> Kernel cmdline - boot arguments to control the system's functionality

Good, and how about "Kernel cmdline - boot arguments to control system functionality"?

> I "insist" on "the", because I don't think it's correct to write `control system's functionality` here. I might be wrong about this.

In regular English, I'd absolutely put a "the" there. But man page titles use a specific flavor of English, one that is brief and terse and straight to the point.

Reply:
Got it, will change it accordingly :)